packages/react-native-sdk/docusaurus/docs/reactnative/01-setup/02-installation/02-expo.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: Installation in Expo
+description: Install the SDK in Expo Development Builds
+---
+
+Our SDK is not available on Expo Go due to native code being required, but you can use the [expo-dev-client](https://docs.expo.dev/development/create-development-builds/) library to run your Expo app with a development build.
+
+## Development Build
+
+If you haven't already, prepare your project for [expo development builds](https://docs.expo.dev/develop/development-builds/installation/).
+
+## SDK Installation
+
+Add the SDK and its required dependencies to your project:
+
+```bash title=Terminal
+npx expo install @stream-io/video-react-native-sdk @stream-io/react-native-webrtc \
+   @config-plugins/react-native-webrtc \
+   react-native-incall-manager react-native-svg \
+   @react-native-community/netinfo @notifee/react-native
```

Comment:
```suggestion
npx expo install @stream-io/video-react-native-sdk
npx expo install @stream-io/react-native-webrtc
npx expo install @config-plugins/react-native-webrtc
npx expo install react-native-incall-manager
npx expo install react-native-svg
npx expo install @react-native-community/netinfo
npx expo install @notifee/react-native
```

This is more of personal preference. Its good as is, but if you would like, I think explicit line per dependency makes it easier to read.